### PR TITLE
Delete all notifications active notifications

### PR DESCRIPTION
**Pull request by jaapmarcus**
**Before:**
image

**After:**
image

### CODE REVIEW

**Comment by AlecRust** [comment]
`package-lock.json` should be removed.

File: bin/v-delete-user-notification [hunk 1]
```diff
@@ -37,21 +41,26 @@ check_hestia_demo_mode
...
-# Deleting notification
-sed -i "/NID='$id' /d" $USER_DATA/notifications.conf 2> /dev/null
-
-# Checking last notification
-if [ -e "$USER_DATA/notifications.conf" ]; then
-	if [ -z "$(grep NID= $USER_DATA/notifications.conf)" ]; then
-		notice='no'
-	fi
-	if [ -z "$(grep "ACK='no'" $USER_DATA/notifications.conf)" ]; then
+# TODO: Delete all if id is "all"
```
Comment (AlecRust) [inline bin/v-delete-user-notification]:
```suggestion
```

**Review by AlecRust — APPROVED** [review]

**Comment by jaapmarcus** [comment]
> `package-lock.json` should be removed.

Then we should add it to .gitignore...

**Comment by tlcd96** [comment]
add also:

```.gitignore
#development on active host
conf
data
log
nginx
php
ssl
web/fm
web/rrd
web/inc/vendor
```

**Comment by jaapmarcus** [comment]
Made a few small changes to bring the behaviour with reload.

**Review by AlecRust — APPROVED** [review]

**Pull request merged**